xrmocap/ops/fourd_association/fourd_associator.py

```diff
@@ -0,0 +1,185 @@
+# yapf: disable
+import logging
+import numpy as np
+from typing import List, Tuple, Union
+import json
+import cv2
+from xrprimer.data_structure.camera import (
+    FisheyeCameraParameter, PinholeCameraParameter,
+)
+from xrprimer.utils.log_utils import get_logger
+
+from xrmocap.data_structure.keypoints import Keypoints
+
+from xrmocap.ops.triangulation.builder import (
+    BaseTriangulator, build_triangulator,
+)
+
+from xrmocap.ops.fourd_association.matching.builder import build_matching
+# yapf: enable
+
+
+class Camera():
+    def __init__(self,cam_param) -> None:
+        super().__init__()
+        self.originK = np.zeros((3,3), dtype=np.float)
+        self.cvR = np.zeros((3,3), dtype=np.float)
+        self.cvT = np.zeros(3, dtype=np.float)
+
+        self.originK = cam_param.intrinsic33()
+        self.cvT = np.array(cam_param.get_extrinsic_t())
+        self.cvR = np.array(cam_param.get_extrinsic_r())
+        self.imgSize = [cam_param.width, cam_param.height]
```

Comment:
`imgSize` doesn't look like a variable defined by this repo, it looks like code copied from somewhere else. If we do need such a var, rename it to img_size.

Reply:
In this classs there are several variables and functions share the same question.

Reply:
I will resolve it